packages/attention/stories/Attention.stories.tsx

```diff
@@ -61,14 +62,16 @@ export function Highlight() {
       <div>
         <div ref={targetEl} className="w-2/3">
           <Box info>
-            <h1>I am a box full of info</h1>
+            <h1>I am a box full with lots of info to show you how the attention element will change its placement in order to stay in view.</h1>
           </Box>
         </div>
         <Attention 
           highlight
-          placement='top'
+          placement='right-start'
           isShowing={show}
           targetEl={targetEl}
+          flip={true}
+          fallbackPlacements={['bottom-end']}
```

Comment:
Should bottom-end show the arrow on the right side maybe? This is what I see when the fallback is executed:
<img width="911" alt="Screenshot 2024-03-08 at 13 29 40" src="https://github.com/warp-ds/react/assets/41303231/6824c391-ec95-4208-a61a-34bddd40dc79">

Reply:
Sure! That makes more sense! :)